README.md

```diff
@@ -0,0 +1,91 @@
+# ``AzureAppConfiguration``
+
+The AzureAppConfiguration package allows for simple construction of
+a `URLRequest` for Azure's App Configuration Service, and decoding
+the response from this service into a dictionary.
+
+A discussion of what Azure App Configuration can be found [here](https://docs.microsoft.com/en-us/azure/azure-app-configuration/overview).
+
+## Overview
+
+The AzureAppConfiguration structure contains a pair of static helper
+functions; no instantiation of an AzureAppConfiguration structure is
+needed or helpful.
```

Comment:
I don't have a super strong opinion either way, but technically we could make `AzureAppConfiguration` an `enum` instead of a `struct` so that the compiler can enforce the inability to instantiate it.

Reply:
Thank you for this. Done.